src/tutorials/0006-anatomy-of-a-cid/06.md

```diff
@@ -20,7 +20,7 @@ Looking at the results from the CID Inspector tool we can see several parts that
 - `Human Readable CID`: breaks down each part of the CID to be easily readable by us humans
 - `Multibase`: `code` is the identifier of the base, in this case `b` for `base32`.
 - `Multicodec`: `code` is the identifier of the codec, in this case `0x70` for `dag-pb`, an IPLD format
-- `Multihash`: breakdown of the multihash into the hashing algorithm used (`18` is the code for `sha2-256`) and the length of the hash (256 bits, which equates to 32 bytes)
+- `Multihash`: breakdown of the multihash into the hashing algorithm used (`18` is the code for `sha2-256`), the length of the hash (256 bits, which equates to 32 bytes) and the content hash itself (digest hex).
```

Comment:
We're using Oxford commas throughout, so need one before the "and" at the end of this list. (Sorry, I've either lost the insert a suggestion button or my mind.)

Reply:
fixed 👍